docs/docs/07-forms.md

```diff
@@ -53,12 +53,16 @@ This will render an input that always has a value of `Hello!`. Any user input wi
     this.setState({value: event.target.value});
   },
   render: function() {
-    var value = this.state.value;
-    return <input type="text" value={value} onChange={this.handleChange} />;
+    return (
+      <input
+        type="text"
+        value={this.state.value}
+        onChange={this.handleChange} />
```

Comment:
Style nit: please put the `/>` on a new line indented to match the opening `<` (that's our style and should be what we've done elsewhere in the docs).